easybuild/easyblocks/m/motioncor2.py

```diff
@@ -91,12 +101,16 @@ def install_step(self):
         """
 
         if (LooseVersion(self.version) >= LooseVersion("1.4")):
-            matches = glob.glob(os.path.join(self.builddir, '%s*' % self.motioncor2_bin))
+            pattern1 = os.path.join(self.builddir, '%s*' % self.motioncor2_bin)
+            pattern2 = os.path.join(self.builddir,
```

Comment:
Maybe this deserves a comment: it seems like you're expecting to find the binary either directly in the build directory, or in a subdirectory named `<name>_<version>`?

Do you have any idea in which case it's in a subdirectory exactly?

Reply:
I just tried to build MotionCor2 and the binary is in a subdirectory in the zipball distributed on https://emcore.ucsf.edu/ucsf-software. As far as I can tell this has changed since v1.4.0, for older version the binary was *not* in a subdirectory.

Reply:
1.4.2 doesn't have a subdir so this needs more specified version checking.

Reply:
hum right... However 1.4.0 does have a subdirectory.
Then I would say @stefan-wolfsheimer was right checking both directly in the build directory and in the subdirectory.